src/main/java/edu/hw1/Task0.java

```diff
@@ -0,0 +1,17 @@
+package edu.hw1;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+final class Task0 {
+
+    private Task0() {
+
+    }
+
+    private final static Logger LOGGER = LogManager.getLogger();
```

Comment:
Есть очень душная спецификация, что правильно писать 
```suggestion
    private static final Logger LOGGER = LogManager.getLogger();
```

Нужно будет везде исправить

Reply:
я это взял из примеров с репозитория, но хорошо, запомню!